package.json

```diff
@@ -38,7 +38,7 @@
     "cross-env": "^5.1.3",
     "css-loader": "^3.4.1",
     "file-loader": "^1.1.6",
-    "node-sass": "^4.13.1",
+    "node-sass": "^6.0.0",
```

Comment:
Can you undo this change? I know we are using a very outdated version here. But this change breaks support with the minimum node version set in this same `package.json` file.

We will eventually upgrade all this dependencies, but we should do it in a separate review.

Feel free to work on that if you think it will make any further change easier for you!